File: src/codes/Users_Manual/geometry_2.cs
~using System;
~using AUTD3Sharp;
~using AUTD3Sharp.Utils;
~using static AUTD3Sharp.Units;
Controller.Builder([
   new AUTD3(Vector3.Zero),
   new AUTD3(new Vector3(AUTD3.DeviceWidth, 0, 0))
         .WithRotation(EulerAngles.ZYZ(0 * rad, MathF.PI / 2 * rad, 0 * rad))
])
~;

File: src/codes/Users_Manual/geometry_2.py
~import numpy as np
~from pyautd3 import AUTD3, Controller, EulerAngles, rad
Controller.builder(
    [
        AUTD3([0.0, 0.0, 0.0]),
        AUTD3([AUTD3.DEVICE_WIDTH, 0.0, 0.0]).with_rotation(
            EulerAngles.ZYZ(0 * rad, np.pi / 2 * rad, 0 * rad)
        ),
    ],
)
